updating links from fwlink to real link if exists - contributes to #3391 (#17053)
dotnet/docs: docs/framework/data/adonet/sql/table-valued-parameters.md

  
|Resource|Description|  
|--------------|-----------------|  

> |[Table-Valued Parameters (Database Engine)](https://docs.microsoft.com/previous-versions/sql/sql-server-2008/bb510489(v=sql.100)) in SQL Server Books Online|Describes how to create and use table-valued parameters.|  

Reviewer: ```suggestion
|[Use Table-Valued Parameters (Database Engine)](/sql/relational-databases/tables/use-table-valued-parameters-database-engine)|Describes how to create and use table-valued parameters.|  
```

> |[User-Defined Table Types](https://docs.microsoft.com/previous-versions/sql/sql-server-2008/bb522526(v=sql.100)) in SQL Server Books Online|Describes user-defined table types that are used to declare table-valued parameters.|  

Reviewer: ```suggestion
|[User-Defined Table Types](https://docs.microsoft.com/previous-versions/sql/sql-server-2008/bb522526(v=sql.100))|Describes user-defined table types that are used to declare table-valued parameters.|  
```

- Use the `bcp` utility program or the <xref:System.Data.SqlClient.SqlBulkCopy> object to load many rows of data into a table. Although this technique is very efficient, it does not support server-side processing unless the data is loaded into a temporary table or table variable.  
  
## Creating Table-Valued Parameter Types  

>  Table-valued parameters are based on strongly-typed table structures that are defined by using Transact-SQL CREATE TYPE statements. You have to create a table type and define the structure in SQL Server before you can use table-valued parameters in your client applications. For more information about creating table types, see [User-Defined Table Types](https://docs.microsoft.com/previous-versions/sql/sql-server-2008/bb522526(v=sql.100)) in SQL Server Books Online.  

Reviewer: ```suggestion
 Table-valued parameters are based on strongly-typed table structures that are defined by using Transact-SQL CREATE TYPE statements. You have to create a table type and define the structure in SQL Server before you can use table-valued parameters in your client applications. For more information about creating table types, see [User-Defined Table Types](https://docs.microsoft.com/previous-versions/sql/sql-server-2008/bb522526(v=sql.100)).  
```
